yaml_specs/modulemd_packager_v2.yaml

```diff
@@ -0,0 +1,271 @@
+# Document type identifier
```

Comment:
Perhaps a line in here somewhere that states what this document type represents--and that it is a simplified subset of the full 'modulemd' type?

Actually, none of the existing specs seem to contain a self-explanatory description of what they represent. They just jump right into detailed descriptions of each individual key.

Reply:
Thank you for adding the self-descriptions in each of the specs!